zeppelin-web/src/app/notebook/notebook.html

```diff
@@ -161,7 +162,9 @@ <h4 class="plus-sign">&#43; Add Paragraph</h4>
          ng-hide="currentParagraph.config.tableHide && viewOnly"
          ng-dblclick="paragraphOnDoubleClick(currentParagraph.id);">
     </div>
-    <div class="new-paragraph last-paragraph" ng-click="insertNew('below');" ng-hide="!$last || viewOnly || asIframe || revisionView">
+    <div class="new-paragraph last-paragraph" ng-click="insertNew('below');"
+         ng-if="!isSequentialRun()"
```

Comment:
shouldn't this be using ng-class?

Reply:
Having looked again at the code, I decided that it would be better to transfer the verification to the ng-hide. Do you think this is the best solution?